pkg/webreg/webreg.go

```diff
@@ -827,6 +872,72 @@ func setWorkflowGraph(t *template.Template, chains registry.ChainByName, workflo
 		})
 }
 
+type environmentLine struct {
+	Documentation string
+	Default       *string
+	Steps         []string
+}
+
+type environmentData struct {
+	Items map[string]environmentLine
+	Type  string
+}
+
+func getEnvironmentDataItems(worklist []api.TestStep, registryRefs registry.ReferenceByName, registryChains registry.ChainByName) map[string]environmentLine {
+	data := map[string]environmentLine{}
+
+	add := func(name, documentation, step string, defaultVal *string) {
+		if data == nil {
```

Comment:
You can omit this now, since it's never going to be `nil` based on how you initialize it :)

Reply:
oh, duh, yeah